### PR TITLE
Fix: Set privileged attribute when populating desired state in tf integration

### PR DESCRIPTION
**Pull request by lechuk47**
Missing part of https://github.com/app-sre/qontract-reconcile/pull/2569

### CODE REVIEW

**Review by bkez322 — APPROVED** [review]
LGTM

File: reconcile/terraform_resources.py [hunk 1]
```diff
@@ -606,6 +606,7 @@ def populate_desired_state(
                 resource_type=oc_resource.kind,
                 name=spec.output_resource_name,
                 value=oc_resource,
+                privileged=spec.namespace.get("clusterAdmin", False),
```
Comment (geoberle) [inline reconcile/terraform_resources.py:609]: when `clusterAdmin` is not defined in a namespace file, the graphql query will return null for it and `get("clusterAdmin", False)` will result in None. 

```suggestion
                privileged=spec.namespace.get("clusterAdmin") or False,
```

Comment (lechuk47) [reply reconcile/terraform_resources.py]: You are right. Even though the result is the same is better set this way. 
Fixed.

Comment (geoberle) [reply reconcile/terraform_resources.py]: absolutely. but explicit types are always nice :)
thanks for fixing it!

**Pull request merged**